### PR TITLE
Style TOTP enrollment box

### PR DESCRIPTION
**Pull request by arbulu89**
# Description
Style TOTP enrollment box.
The unique small addition is the `Cancel` button, which just closes the box.

styled_totp_box

@jagabomb I have not added the `(TOTP)` text in the label, as it is already mentioned in the info icon box, and it totally messes up the whole alignment of the box. I hope it works this way.

### CODE REVIEW

File: assets/js/pages/Profile/ProfileForm.jsx [hunk 1]
```diff
@@ -124,31 +125,44 @@ function ProfileForm({
...
-              <TotpEnrollementBox
-                errors={errors}
-                qrData={totpQrData}
-                secret={totpSecret}
-                loading={loading}
-                verifyTotp={onVerifyTotp}
+
+          <Label
+            className="col-start-1 col-span-1"
+            info="Setup a multi factor TOTP authentication besides your password to increase security
```
Comment (EMaksy) [inline assets/js/pages/Profile/ProfileForm.jsx:131]: I guess there is a typo `multi factor` --> `multifactor`

**Review by EMaksy — APPROVED** [review]
LGTM, just left one comment :+1:

**Comment by jagabomb** [comment]
> I have not added the (TOTP) text in the label, as it is already mentioned in the info icon box, and it totally messes up the whole alignment of the box. I hope it works this way.

**Review by jagabomb — APPROVED** [review]
LGTM

**Pull request merged**